29/btree.c

```diff
@@ -179,7 +190,11 @@ static char *toString(btree_t *btree) {
 }
 
 static char *toStringHelper(node_t *h, int ht, char *indent) {
-  char *s = malloc(1024 * sizeof(char));
+  // avoid weird characters and subsequent weird behavior when free by explicit initialization.
+  char *s = calloc(1024 * sizeof(char),1024);
```

Comment:
Isn't this line should be `calloc(sizeof(char), 1024)`?. I'll review the rest code later, I need some time to refamiliarize the code and data structure...

Reply:
Yes, my mistake. According to the `man calloc`, `void *calloc(size_t nmemb, size_t size);` maybe implies `calloc(1024, sizeof(char))` better.